python/proton/_handlers.py

```diff
@@ -50,6 +50,10 @@ class OutgoingMessageHandler(Handler):
 
     :param auto_settle: If ``True``, settle all messages (default). Otherwise
         messages must be explicitly settled.
+
+        .. note:: Sender auto-settlement only occurs for a delivery after the
```

Comment:
Delete this as a separate para add the comment into the basic description of auto settle behaviour as in:
If  ``True`` (default), automatically settle messages upon receiving a settled disposition for that delivery...

Reply:
Done.